src/Symfony/Bridge/ProxyManager/Tests/LazyProxy/Fixtures/includes/foo.php

```diff
@@ -2,9 +2,18 @@
 
 class ProxyManagerBridgeFooClass
 {
-    public $foo, $moo;
+    public $foo;
+    public $moo;
 
-    public $bar = null, $initialized = false, $configured = false, $called = false, $arguments = array();
+    public $bar = null;
+
+    public $initialized = false;
+
```

Comment:
This LF is "inherited" from the original separator between the property declaration and the next class element. I'll add a test for this behavior.